Neos.SiteKickstarter/Resources/Private/AfxGenerator/Fusion/Document/AbstractPage.fusion

```diff
@@ -0,0 +1,58 @@
+prototype({packageKey}:Document.AbstractPage) < prototype(Neos.Neos:Page) {
+  head {
+    headTags = Neos.Fusion:Component {
+      renderer = afx`
+        <meta http-equiv="x-ua-compatible" content="ie=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      `
+    }
+
+    titleTag = Neos.Fusion:Tag {
+      tagName = 'title'
+      content = ${q(node).property('title')}
+    }
+
+    # Add stylesheets to your page
+    stylesheets.site = Neos.Fusion:Tag {
+      tagName = 'link'
+      attributes.rel = 'stylesheet'
+      attributes.href = Neos.Fusion:ResourceUri {
+        # Change to your file path
+        path = 'resource://{packageKey}/Public/site.css'
+      }
+    }
```

Comment:
```suggestion
     stylesheets.site = afx`
        <link rel="stylesheet" href={StaticResource.uri('{packageKey}', 'Public/site.css')} />
    `
```
Since we have now an [eel helper for static sources](https://github.com/neos/neos-development-collection/pull/3134), we could make use of this.